benchmarks/asv/optimize.py

```diff
@@ -42,60 +42,54 @@ def create_sampler(sampler_mode: str) -> BaseSampler:
 
 
 class OptimizeSuite:
-    def objective(self, trial: optuna.Trial) -> float:
+    def objective(self, trial: optuna.Trial, n_objectives: int) -> list[float]:
```

Comment:
I guess what makes this function lack of readability is the extra arg of `n_objectives`.
How about the following code for this function.
```python
    def objective(self, trial: optuna.Trial) -> tuple[float, float, float]:
        x = trial.suggest_float("x", -100, 100)
        y = trial.suggest_int("y", -100, 100)
        return x**2 + y**2, (x - 2) ** 2 + (y - 2) ** 2, (x + 2) ** 2 + (y + 2) ** 2
```
Then, use it like this:
```python
study.optimize(lambda trial: self.objective(trial)[:n_objectives], n_trials=n_trials)
```

Reply:
As in https://github.com/optuna/optuna/pull/5635/files#r1731981221, I think it would be better just to adding another 3 dimensional objective function for readability and maintenanceability.